<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loginview</class>
 <widget class="QMainWindow" name="Loginview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢迎使用</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>240</width>
      <height>210</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>561</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabLogin">
     <attribute name="title">
      <string>登录</string>
     </attribute>
     <widget class="QLabel" name="hostname_login">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>主机</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="iHostname_login">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>35</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
     <widget class="QLabel" name="username_login">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="iUsername_login">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>65</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_login">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="iPassword_login">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>95</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRegister">
     <attribute name="title">
      <string>注册</string>
     </attribute>
     <widget class="QLabel" name="hostname_reg">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>主机</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="iHostname_reg">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>35</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
     <widget class="QLabel" name="username_reg">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="iUsername_reg">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>65</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_reg">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="iPassword_reg">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>95</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="regButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="aboutLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>关于我们</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
